test/Faker/Provider/InternetTest.php

```diff
@@ -117,4 +117,50 @@ public function testMacAddress()
     {
         $this->assertRegExp('/^([0-9A-F]{2}[:]){5}([0-9A-F]{2})$/i', Internet::macAddress());
     }
+
+    public function test_htmlLoremDefault()
+    {
+        $html = Internet::html();
+        $this->assertTrue($this->containsHtml($html));
+    }
+
+    public function test_htmlLoremSmallLengths()
+    {
+        $plus5 = Internet::html(Internet::SMALLEST_HTML_LENGTH+5);
+        $this->assertTrue($this->containsHtml($plus5));
+    }
+
+    public function test_htmlTooSmall()
+    {
+        $this->setExpectedException('InvalidArgumentException');
+        Internet::html(Internet::SMALLEST_HTML_LENGTH + 4);
+        Internet::html(Internet::SMALLEST_HTML_LENGTH - 1);
+    }
+
+    public function test_htmlMinimal()
+    {
+        $this->setExpectedException('InvalidArgumentException');
+        Internet::html(Internet::SMALLEST_HTML_LENGTH - 1);
+    }
+
+    public function test_htmlFibonacci()
```

Comment:
It tests the provider over large numbers of text argument (using a Fibonacci suite), but it does not check that these various text sizes actually match the various cases of the `switch($maxLength)` operator of the Internet::html() implementation. So all this remains quite useless for now.
=> Removed!
Thanks for having spotted this.